src/v2/port_mapping.in.rs

```diff
@@ -145,25 +185,40 @@ impl fmt::Display for PortMapping {
         if let Some(ports) = self.host_ports {
             write!(f, "{}:", ports)?;
         }
-        write!(f, "{}", self.container_ports)
+        write!(f, "{}/", self.container_ports)?;
+        write!(f, "{}", self.protocol)
```

Comment:
Maybe we want to skip the `/` and the protocol if `self.protocol.is_default()`? This might help avoid confusing downstream tools that don't understand UDP.

Reply:
Will do.